juniper/src/vespa/juniper/sumdesc.cpp

```diff
@@ -29,6 +33,30 @@ bool wordchar(const unsigned char* s)
     }
 }
 
+bool wordchar_or_il_ann_anchor(const unsigned char* s)
+{
+    unsigned char c = *s;
```

Comment:
Please move the content of this function into a helper function that both wordchar_or_il_ann_anchor() and wordchar_or_il_ann_terminator() can call.